packages/descendant/src/use-descendant.ts

```diff
@@ -1,5 +1,5 @@
 import { createContext, mergeRefs } from "@chakra-ui/react-utils"
-import { RefCallback, useRef, useState } from "react"
+import React, { RefCallback, useRef, useState } from "react"
```

Comment:
Oh wait, is it because of this is only a `.ts` file? So that means it is not necessary in here, I guess.
Saw that on a few more files you changed, then better remove it.

Reply:
it was indicated by webstorm that it is referring to a global, and it's better to import it.